src/lib/helpers.ts

```diff
@@ -33,7 +50,9 @@ export class utils {
             const inheritorEncKey = await creator.idx?.ceramic.did?.createDagJWE(aesKey, [inheritorDid]);
```

Comment:
Let's rename the `inheritor` to `beneficiary` everywhere according to our glossary: https://www.notion.so/Safient-concepts-and-glossary-754a015f132642ce8e915d7e95fe2f0b